l10n_be_partner_kbo_bce/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "license": "AGPL-3",
     "author": "Noviat,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/l10n-belgium",
-    "depends": ["partner_identification", "base_vat_sanitized"],
+    "depends": ["partner_identification"],
```

Comment:
It seems `_fix_vat_number` is in `base_vat`. Should it not depend on it ? @luc-demeyer -de